.github/actions/restore_artifacts/action.yml

```diff
@@ -0,0 +1,58 @@
+name: 'Restore artifacts'
+description: 'Take artifacts from a shared drive and load them back to GitHub'
+inputs:
+  storage_dir:
+    description: "Subdirectory name for artifacts, same as product type"
+    required: true
+  storage_root:
+    description: "Root path of the artifacts storage"
+    required: true
+  event_name:
+    description: "GitHub event name. If not set - taken from github context"
+    required: false
+  branch_name:
```

Comment:
not clear why do we need to set branch name, for case when we restore the latest available artifact?

Reply:
Latest available _from branch_